lib/styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import kebab from "kebab-case";
+
+export function styledDefaults(defaults) {
+  return (props) =>
+    Object.entries(defaults)
+      .map(([key, value]) => `${kebab(key)}: ${props[key] || value};`)
+      .join("\n");
+}
```

Comment:
**Nice one. Very advanced but also a little over engineered. ☄️**
It could be that I have tomatoes on my eyes 🍅🍅 but I don't see where this function is used - so imported and called. 

If you do not use this function, please delete this file. 🧹